### DISCUSSION TITLE
how to disable ``http://host`` auto redirect to ``http://host/account/sign-in?redirect_url=%2Fharbor%2Fprojects``

### DISCUSSION THREAD

**Discussion by yahuian** [goharbor/harbor#16939] [Q&A]
for security, i don't want to expose my habor login page, but now habor will auto redirect when i only enter host url.
can i change it? :smiley:

**Comment by AllForNothing** [comment]
@YahuiAn   The default url/page requires auth,  so it will redirect to login page.  So
1.   you can log in first and then open the host url;
2.   Directly open the page that does not require auth,  like `https://${host}/harbor/projects/${projectId}/repositories/${repoNmae}/artifacts-tab?publicAndNotLogged=yes`
Note: the project must be public

**Reply by yahuian** [reply]
thanks for your reply, but i don't want someone else know my harbor url/page to prevent password blasting.
